src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ModuleInitializerAttribute.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Used to indicate to the compiler that a method should be called
+    /// in its containing module's initializer.
+    /// </summary>
+    /// <remarks>
+    /// When one or more valid methods
+    /// with this attribute are found in a compilation, the compiler will
+    /// emit a `&lt;Module&gt;..cctor` which calls each of the attributed methods.
+    ///
+    /// Certain requirements are imposed on any method targeted with this attribute:
+    /// - The method must be `static`.
+    /// - The method must be an ordinary member method, as opposed to a property accessor, constructor, local function, etc.
+    /// - The method must be parameterless.
+    /// - The method must return `void`.
+    /// - The method must not be generic or be contained in a generic type.
+    /// - The method must be accessible from the `&lt;Module&gt;` class.
+    ///   - This means the method's effective accessibility must be `internal` or `public`.
```

Comment:
```suggestion
    ///  - The method's effective accessibility must be `internal` or `public`.
```
Another reference of `<Module>` class